### DISCUSSION TITLE
Week 14 征集

### DISCUSSION THREAD

**Discussion by rewired-gh** [hitszlug/weekly#61] [Ideas]
Week 14 征集

**Comment by LittleYe233** [comment]
## 本周看了啥

**Tailwind CSS 框架**

https://tailwindcss.com

将大部分 CSS 样式转为 class 描述，并将一些常见的样式搭配包装成独立的 class，简化开发。

安装简便，覆盖面广，对于不依赖其他框架的 simple website 和含模板系统（如 ejs）的网站也适用。

缺点可能是丧失了一定的自由度，比如若要控制宽度为 12.3456...% 就要靠牺牲 elegance 来实现..

个人反思：以往手敲 CSS、借助 JS hack 样式的时代必须要过去了。当然基础的 CSS 也是必须要学习的，否则任何层层包装的前端框架都救不了网站的 UI。

**Reply by shirok1** [reply]
有没有看过 https://windicss.org/

**Reply by LittleYe233** [reply]
> 有没有看过 https://windicss.org/

很遗憾并没有（

**Reply by rewired-gh** [reply]
> 有没有看过 https://windicss.org/

There's no significant advantage from Tailwind, especially considering Tailwind JIT has already been a thing; although it truly is more modern than Tailwind in some aspects of foundational design.

**Reply by rewired-gh** [reply]
Despite Tailwind is a capable and excellent framework, there are tons of brilliant web frontend frameworks. So I'm afraid it might not be suitable to introduce a commonly used framework in LUG Weekly, but it could be introduced in an article related to web frontend development in LUG Wiki.
